ui/main/src/app/authentication/auth-handler.ts

```diff
@@ -94,7 +94,7 @@ export abstract class AuthHandler {
     private decodeToken(token: string): any {
         try {
             return jwt_decode(token);
-        } catch (Error) {
+        } catch (err) {
```

Comment:
Ajouter une ligne de log : this.logger.error(" Error in token decoding " + err);